src/pivot-table/hooks/__tests__/use-column-width.test.ts

```diff
@@ -181,6 +182,7 @@ describe("useColumnWidth", () => {
       expect(getRightGridColumnWidth(1)).toBe(ColumnWidthValues.AutoMin);
       expect(getRightGridColumnWidth(2)).toBe(ColumnWidthValues.AutoMin);
     });
+
```

Comment:
Seems like the Left grid is missing a test case where the icon width is included?

Reply:
They are tested, it was already taking icon into account for the left side, before this pr